### ISSUE TITLE
Replace pre-trained models with deprecated embeddings

### ISSUE THREAD

**Issue by alanakbik** [zalandoresearch/flair#974]
There's a list of pre-trained models in Flair that still use the deprecated `CharLMEmbeddings` and may suffer from the dropout bug. These models are (_x_ marks dropout bug) and need to be retrained: 

`SequenceTagger`
- [x] 'ner-fast' (x)
- [x] 'ner-ontonotes'
- [x] 'ner-ontonotes-fast'
- [x] 'pos-fast' (x)
- [x] 'frame' (x)
- [x] 'frame-fast' (x)
- [x] 'chunk-fast' (x)
- [x] 'de-pos' (x)
- [x] 'de-ner'
- [x] 'fr-ner'

`TextClassifier`
- [x]  'en-sentiment'
- [x] 'de-offensive-language'
